### PR TITLE
chore(package): upgrade minim to v0.22.1

### PR DESCRIPTION
**Pull request by opichals**

### CODE REVIEW

File: src/Components/Attributes/test/Attributes.js [hunk 1]
```diff
@@ -61,10 +61,10 @@ describe('Attributes', () => {
     }, { id: 'User' });
 
     const dataStructures = new Category(
-      new minim.ArrayElement([
```
Comment (opichals) [inline src/Components/Attributes/test/Attributes.js]: @kylef Could you please check this? I had to change this for the tests to pass. To me it looks like minim API breaking change.

Comment (kylef) [reply src/Components/Attributes/test/Attributes.js]: That's odd, while I think what you've changed is more correct. Given the API Elements spec, a category contains an array, not an ArrayElement so what you had before wasn't really correct semantically. Although given ArrayElement and Array have similiar interfaces it likely just happened to work.

---

I am going to take a guess that you was seeing an error thrown `Cannot set content to given value` which I think may be caused due to multiple minim / minim namespaces at play. There is a dev-dependecy on Fury (`"fury": "=3.0.0-beta.6"`) which will bring an older version of Minim around. I would expect that will cause problems like this. You shouldn't be mix and matching minim versions and namespaces in the same minim element tree. However this particular code path seems to use the same underlying Minim version (albeit `minim.ArrayElement` vs using `Category` from `minimNamespace`) so I am not sure if that is exactly what is at play here, it might be that some test using this element tree is failing after adding a different element into the tree or similiar.

I think it would be a good idea to update Fury (and the adapters) to the latest version so that there is just one version of minim used and tested. There could be a problem right now because some tested elements could be from a different version of Minim via Fury injected via the tests / dev dependencies in certain code paths.

That does raise a potential API design problem of Attributes Kit, is that it allows you to pass elements from one version of minim / namespace and then uses it with it's own version of Minim (JSON06Serialiser). While it is possible to mix and match the versions of the namespaces and elements (minim 0.22.0 onwards) it can be problematic if you don't provide the namespace into the serialiser. If the versions of minim are the same then this likely would just work, but if they do differ (just could be the case if the integrator of attributes kit has a different version of minim) then it could could broken behaviour so it would perhaps make sense for attributes kit API to allow you to provide a minim namespace in the case that you are passing in a minim element tree so it can deserialise with the same namespace.

---

So tl;dr / action points:

- [ ] Update Fury to latest version

---

Something to think about / potentially for the future:

- Alter Attributes Kit to require a namespace to be injected into it when you are passing a minim element to prevent potential compatibility issues.
    - Second though, perhaps the API shouldn't accept a minim element and the caller should just deal with the deserialisation.

**Comment by opichals** [comment]
Closing in favor of https://github.com/apiaryio/attributes-kit/pull/481 which addresses the `fury` toolchain upgrade as well.
